### PR TITLE
RDS Rule11-6.md

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: docs/section11/Rule11-6.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# HVAC_SystemZoneAssignment - Rule 11-6
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-6  
+**Rule Description:** Piping losses shall not be modeled. 
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, f
```
Comment (claperle) [inline docs/section11/Rule11-6.md:8]: I think this is i not f

Comment (claperle) [inline docs/section11/Rule11-6.md:8]: This has a differnt App G reference dependign on whether you are referring to the baseline or proposed design model. It looks like this check is for the baseline model whch would be Table G3.1 #11, baseline column, i.

Is there a separate RDS for checking the proposed?

File: docs/section11/Rule11-6.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+**Evaluation Context:** Each SWH space type
```
Comment (claperle) [inline docs/section11/Rule11-6.md:10]: Should this be Each SWH BAT?

File: docs/section11/Rule11-6.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+**Rule Description:** Piping losses shall not be modeled. 
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, f
+
+**Evaluation Context:** Each SWH space type  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_connected_to_use_type()
+2. get_SHW_types_and_SHW_use()
```
Comment (claperle) [inline docs/section11/Rule11-6.md:15]: Are these function names outdated?

File: docs/section11/Rule11-6.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+**Applicability Checks:**
+- Each SHW use type is applicable if there are SHW loads
+- call the function get_SHW_equipment_connected_to_use_type: `shw_equip_dict = get_SHW_equipment_connected_to_use_type(B_RMD)`
+- also use get_SHW_types_and_SHW_use to get the SHW use types and SHW use in the building: `shw_types_and_use_dict = get_SHW_types_and_SHW_use(P_RMD)`
```
Comment (claperle) [inline docs/section11/Rule11-6.md:20]: was the name of this function changed? I had thought the names were change to make it more clear that SHW types means SHW BAT

File: docs/section11/Rule11-6.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- look through each of the SHW types in the building: `for shw_use_type in shw_types_and_use_dict:`
+  - check if there are SHW loads in this use type, continue to rule logic: `if len(shw_types_and_use_dict[shw_use_type]) > 0: RULE_APPLICABLE`
+  - otherwise, rule is not applicable for this shw_use_type: `else: NOT_APPLICABLE`
+
+
+    ## Rule Logic:
+    - create a value indicating whether piping losses were modeled: `piping_losses_modeled = false`
+    - look at every ServiceWaterPiping connected to the shw_use_type: `for piping_id in shw_types_and_use_dict[shw_use_type]["Piping"]`
```
Comment (claperle) [inline docs/section11/Rule11-6.md:28]: Suggest language change in this RDS similar to others to make it clear that SWH type means SHW BAT

File: docs/section11/Rule11-6.md (hunk 1 above)
Comment (KarenWGard) [reply docs/section11/Rule11-6.md:8]: thanks.  Yes.  11-7 refers to the proposed.

File: docs/section11/Rule11-6.md (hunk 2 above)
Comment (KarenWGard) [reply docs/section11/Rule11-6.md:10]: yes

File: docs/section11/Rule11-6.md (hunk 3 above)
Comment (KarenWGard) [reply docs/section11/Rule11-6.md:15]: yes.  updated

File: docs/section11/Rule11-6.md (hunk 4 above)
Comment (KarenWGard) [reply docs/section11/Rule11-6.md:20]: yes.  updated

File: docs/section11/Rule11-6.md (hunk 5 above)
Comment (KarenWGard) [reply docs/section11/Rule11-6.md:28]: updated

File: docs/section11/Rule11-6.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+# HVAC_SystemZoneAssignment - Rule 11-6
+
+**Schema Version:** 0.0.36
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-6.md:3]: Schema is currently on 0.0.37. Not sure if any of the updates impact this RDS.

File: docs/section11/Rule11-6.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# HVAC_SystemZoneAssignment - Rule 11-6
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-6  
+**Rule Description:** Piping losses shall not be modeled. 
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, i
+
+**Evaluation Context:** Each SWH BAT
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-6.md:10]: each SWH BAT or each ServiceWaterHeatingDistributionSystem?

File: docs/section11/Rule11-6.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_associated_with_each_SWH_bat()
+
+**Applicability Checks:**
+- Each SHW use type is applicable if there are SHW loads
+- also use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW BATs and SHW use in the building: `shw_bats_and_use_dict = get_SHW_equipment_associated_with_each_SWH_bat(P_RMD)`
+- look through each of the SHW BATs in the building: `for shw_bat in shw_bats_and_use_dict:`
+  - check if there are SHW loads in this use type, continue to rule logic: `if len(shw_bats_and_use_dict[shw_bat]) > 0: RULE_APPLICABLE`
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-6.md:20]: Not clear why ServiceWaterHeatingUse is required for this to be applicable. What is the scenario that this prevents? Couldn't `ServiceWaterPiping.piping_losses_modeled` be defined as true without ServiceWaterHeatingUse being defined?

File: docs/section11/Rule11-6.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  - otherwise, rule is not applicable for this shw_bat: `else: NOT_APPLICABLE`
+
+
+    ## Rule Logic:
+    - create a value indicating whether piping losses were modeled: `piping_losses_modeled = false`
+    - look at every ServiceWaterPiping connected to the shw_bat: `for piping_id in shw_bats_and_use_dict[shw_bat]["Piping"]`
+      - get the piping: `piping = get_component_by_id(piping_id, B_RMD, ServiceWaterPiping) `
+      - check if the piping has piping_losses_modeled: `if piping.piping_losses_modeled:`
+        - set the piping_losses_modeled to true: `piping_losses_modeled = true`
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-6.md:29]: can proceed to rule assertion as soon as 1 piping has losses modeled

**Review by JacksonJ-KC — CHANGES_REQUESTED** [review]

**Review by claperle — APPROVED** [review]

File: docs/section11/Rule11-6.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
+# Service Water Heating - Rule 11-6
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-6  
+**Rule Description:** Piping losses shall not be modeled.
```
Comment (claperle) [inline docs/section11/Rule11-6.md:6]: I think we should have a rule that checks this for the baselien and another that checks the proposed

Comment (KarenWGard) [reply docs/section11/Rule11-6.md:6]: This is covered in the global baseline versus proposed check

Comment (JacksonJ-KC) [reply docs/section11/Rule11-6.md:6]: This needs to be changed to true if you want this to be covered by rule 1-6.

https://github.com/open229/ruleset-model-description-schema/blob/develop/schema-source/ASHRAE229_extra.schema.yaml#L5768

Comment (KarenWGard) [reply docs/section11/Rule11-6.md:6]: it's covered here:  lines 101-103

just updated to reflect change to `are_thermal_losses_modeled`

[docs/section11/SHW comparison fields.yaml](url)

Comment (JacksonJ-KC) [inline docs/section11/Rule11-6.md:6]: line 6 needs 2 spaces at the conclusion of the line for md formatting

File: docs/section11/Rule11-6.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+**Rule Description:** Piping losses shall not be modeled. 
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, i
+
+**Evaluation Context:** Each SWH Distribution System  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_swh_components_associated_with_each_swh_distribution_system()
+
```
Comment (JacksonJ-KC) [inline docs/section11/Rule11-6.md:15]: 2. get_component_by_id()

**Review by JacksonJ-KC — APPROVED** [review]

**Pull request merged**